test-id: tests/test_purge_query_plan.py::test_purge_and_recovery_lookups_use_an_index[postgres]
--- a/migrations/01M4SVVY5XE3S18Y0CPQ8TATDP_purge_and_lookup_indexes.sql	2026-10-13 05:35:10.316751931 +0000
+++ b/migrations/01M4SVVY5XE3S18Y0CPQ8TATDP_purge_and_lookup_indexes.sql	2026-10-13 05:35:10.328320424 +0000
@@ -9,9 +9,6 @@
 CREATE INDEX IF NOT EXISTS idx_executions_application_end_time
     ON pipeline_executions (application, end_time) WHERE end_time IS NOT NULL;
 
-CREATE INDEX IF NOT EXISTS idx_queue_messages_execution_id
-    ON queue_messages ((payload ->> 'execution_id'));
-
 CREATE INDEX IF NOT EXISTS idx_queue_dlq_execution_id
     ON queue_messages_dlq ((payload ->> 'execution_id'));
 
